### PR TITLE
Fix old and add new translations

### PR DESCRIPTION
**Pull request by sheilagomes**
Add Bulgarian, Catalan, Czech, German, Greek, French, Italian, Dutch, Polish, Russian, Slovakian, Swedish, Thai and Ukrainian translations. Fix Danish, English, Finnish, Korean, Indonesian and Romanian translations and Crowdin configuration file. Tracked in task [LOC-9916](https://vtex-dev.atlassian.net/browse/LOC-9916).

[LOC-9916]: https://vtex-dev.atlassian.net/browse/LOC-9916?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: messages/en.json [hunk 1]
```diff
@@ -7,19 +7,19 @@
...
   "store/product-list.originalPrice": "Original price",
   "store/product-list.changeTo": "Change to",
-  "store/product-list.manualPricePlaceholder": "Type a monetary value",
-  "store/product-list.applyManualPrice": "ok",
+  "store/product-list.manualPricePlaceholder": "Enter an amount",
+  "store/product-list.applyManualPrice": "OK",
   "store/product-list.quantitySelector.label": "{quantity} {measurementUnit}",
   "store/product-list.quantitySelector.maxValueLabel": "{quantity} {measurementUnit} +",
-  "store/product-list.quantityUnitMultiplierMismatch": "This product is sold by fractions of {unitMultiplier}{measurementUnit}. Given that, the inserted quantity has been rounded to {roundedValue}{measurementUnit}.",
+  "store/product-list.quantityUnitMultiplierMismatch": "This product is sold by fractions of {unitMultiplier}{measurementUnit}. That is why the entered quantity has been rounded to {roundedValue}{measurementUnit}.",
```
Comment (iago1501) [inline messages/en.json:22]: This text is being used in our tests, changing this label breaks some tests, suggest keeping the same translation or/and fixing the text on tests

**Review by iago1501 — CHANGES_REQUESTED** [review]

File: messages/en.json (hunk 1 above)
Comment (iago1501) [reply messages/en.json:22]: https://github.com/vtex-apps/product-list/blob/8bb6492114ed0b78ea17da36d86c51e84600cb04/react/components/__tests__/QuantityStepper.test.tsx#L43

**Review by camilavcoutinho — APPROVED** [review]

File: messages/en.json (hunk 1 above)
Comment (sheilagomes) [reply messages/en.json:22]: Done, @iago1501 !

**Review by iago1501 — APPROVED** [review]
LGTM, thanks for that 👍🏻

**Comment by sheilagomes** [comment]
> LGTM, thanks for that 👍🏻

Great, glad to know it works! Could you please approve and deploy this one, @iago1501 ?

**Pull request merged**